templates/create-daml-app/ui/src/components/LoginScreen.tsx.template

```diff
@@ -61,7 +61,20 @@ const LoginScreen: React.FC<Props> = ({onLogin}) => {
     </Grid>;
 
   const InsecureLogin: React.FC<{auth: Insecure}> = ({auth}) => {
+    const [options, setOptions] = useState([{key: '', text: '', value: ''}]);
     const [username, setUsername] = React.useState('');
+
+    useEffect(() => {
+      (async () => {
+        const token = auth.makeToken('public');
+        const ledger = new Ledger({token: token});
+        const allUserAliases = await ledger.query(User.Alias);
+        const allUsers = await ledger.listUsers();
+        setOptions(allUsers.filter((u) => !['participant_admin', 'public'].includes(u.userId) ).map((u) => ({key: u.userId, text: u.userId, value: u.userId})))
```

Comment:
I think I’m confused, shouldn’t this use `allUseAliases`?

Reply:
There are two options. Either you log in using your userId. Then you want to see all available users via `listUsers`. Or you log in using your alias. However, if you want to log in by alias, you will need to store the userId as well in the Alias contract. Also, there is a possibility that the Alias was created by a user on a different participant and the login will fail.
I implemented the first option. I think the `allUserAliases` was a left over that should be removed by now.

Reply:
I don’t think we should list all users here on the login screen. Getting the list of users requires admin rights which means this isn’t going to work in a properly authorized setup.

Reply:
Would you be fine with a normal input field in that case?

Reply:
Yeah I think a text input on the login screen is perfectly fine.

Reply:
Cool, I'll change it.